WickedEngine/wiRenderer.cpp

```diff
@@ -6030,7 +6060,11 @@ void DrawDebugWorld(
 
 	if (GetRaytraceDebugBVHVisualizerEnabled())
 	{
-		RayTraceSceneBVH(scene, cmd);
+		//PE: Check if debug BVH is possible. or it will crash.
```

Comment:
Path tracing also needs the debug bvh visualizer

Reply:
Path tracing debug have its own call , so it works fine :)